### ISSUE TITLE
No options to hide Shorts in my subscriptions section (STN Beta 15.81)

### ISSUE THREAD

**Issue by kldvb** [yuliskov/SmartTube#1794]
No options to hide Shorts in my subscriptions section

**Comment by Wofflo** [comment]
This would be a nice feature. They are really irritating.

**Comment by popy2k14** [comment]
+1 for subscriptions/channels
I find them so irritating!!

**Comment by rspearson** [comment]
+1 Yes please. This would be fantastic.

**Comment by bfelten1948** [comment]
Absolutely! Please! Leave that crap to the Tik Tok kids.

**Comment by r14ull** [comment]
> Absolutely! Please! Leave that crap to the Tik Tok kids.

You can do this now - its in Settings > General > Misc > Hide Shorts from subscriptions

**Comment by MailYouLater** [comment]
> +1 for subscriptions/channels I find them so irritating!!

Please show your support for hiding shorts in channels in issue #2115.

> You can do this now - its in Settings > General > Misc > Hide Shorts from subscriptions

Yes.  This issue should be closed.
![image](https://user-images.githubusercontent.com/1279475/217721051-9fa33ca5-be4f-4821-aab4-f8db9095cbd9.png)

**Comment by popy2k14** [comment]
@MailYouLater 
Which version do you have?
In the current stable i dont find the "Hide shorts from Subsciptions".
I just have "Hide streams from Subscriptions".

thx

**Comment by MailYouLater** [comment]
"Current" isn't a version.  What version is "current" changes.  Please state the actual version number, as shown in Settings > About.
 
The screenshot was from the current ([17.12](https://github.com/yuliskov/SmartTubeNext/releases/tag/17.12)) beta release, running in an emulator because I wasn't with my TV at the time, however I had already known it was there from seeing it on my TV several updates ago.  And while I apologize for being pedantic earlier in this post, I too had thought that I had the curent stable release installed, but upon closer inspection, I'm currently on [SmartTube 16.70 Stable](https://github.com/yuliskov/SmartTubeNext/releases/tag/16.70s) <sub>(apparently auto-updates are silently failing, I'll have to update manually)</sub>, and (I just checked) the option in question is present.

**Comment by popy2k14** [comment]
Sorry for beeing unclear. My current stable is 16.93 (current stable from Releases section: https://github.com/yuliskov/SmartTubeNext/releases).

No, the option "Hide shorts from subscriptions" is not available in this stable version! Checked yesterday on my two tv's running 16.93 stable.

So will have to wait for the next stable.

**Comment by MailYouLater** [comment]
I just manually updated to version 16.93 stable on my MiBox S with Android TV 9.
![Screenshot](https://user-images.githubusercontent.com/1279475/218043476-9bd3848c-3a14-4ce0-8faf-abcfc9e02732.png)
![Screenshot](https://user-images.githubusercontent.com/1279475/218043555-17cdf962-e2a7-4b45-9ce0-b9996e9054c1.png)
![Screenshot](https://user-images.githubusercontent.com/1279475/218043623-b4516bfe-b47d-477e-a922-6908978dd62d.png)
![Screenshot](https://user-images.githubusercontent.com/1279475/218043765-09a3f5a3-8d2f-47fd-b201-bd402b1ca5d2.png)

Looks like it's there.  If it's really not there for you...  then what model(s) are these TVs, what OS version(s) are they running, any other possibly pertinant info?

**Comment by popy2k14** [comment]
Yeah, you are right, it's there. In german "Kurzvideos in Abos ausblenden". Sorry for bothering you.

Would be also nice for channels, see #2115 .

**Comment by neooblaster** [comment]
Hi @MailYouLater ,

Just a little feedback.
I thought the Option was nos available in version 16.93 but this is a translation issue.
I switch to English and I finaly found the option `Hide shorts from Subscription`.
In french, the settings is displayed as "Masquer les passages de demande d'Abonnements" ("Hide Subscription request passages").
I checked the option and finaly, it's the right settings which is enabled :  `Hide shorts from Subscription`.
If needed, in French the more approriate translation could be `Masquer les shorts des abonnements`

Thanks a lot for your work and your support :)

**Comment by MailYouLater** [comment]
@neooblaster: If the French translation needs to be updated, it should be changed here:
https://github.com/yuliskov/SmartTubeNext/blob/d516b8b874859e888844dce0888b4ebb4c4dc0b0/common/src/main/res/values-fr/strings.xml#L231

@popy2k14: If you weren't finding it before because of a poor German translation, perhaps you could suggest a better one?  
If so, it should be changed here: 
https://github.com/yuliskov/SmartTubeNext/blob/d516b8b874859e888844dce0888b4ebb4c4dc0b0/common/src/main/res/values-de/strings.xml#L232

Do you guys know how to make & submit a pull request with your suggested changes?

**Comment by popy2k14** [comment]
> @neooblaster: If the French translation needs to be updated, it should be changed here:
> 
> https://github.com/yuliskov/SmartTubeNext/blob/d516b8b874859e888844dce0888b4ebb4c4dc0b0/common/src/main/res/values-fr/strings.xml#L231
> 
> @popy2k14: If you weren't finding it before because of a poor German translation, perhaps you could suggest a better one? If so, it should be changed here:
> 
> https://github.com/yuliskov/SmartTubeNext/blob/d516b8b874859e888844dce0888b4ebb4c4dc0b0/common/src/main/res/values-de/strings.xml#L232
> 
> Do you guys know how to make & submit a pull request with your suggested changes?

thx, it's clear.
Just was missing an "Hide shorts in channels" feature. Where an issue is open for that.
sorry for bothering you.

**Comment by hunited** [comment]
Hi, Settings > General > Misc > Hide Shorts from subscriptions not working. I WANT to see shorts, but the checkbox not working, each state hide shorts. Can anyone help me? 16.58, Philips A12 tv.
